src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/pricing/importer/PricesSpreadsheetTest.kt

```diff
@@ -0,0 +1,85 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.pricing.importer
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import org.apache.poi.ss.usermodel.Row
+import org.apache.poi.ss.usermodel.Sheet
+import org.apache.poi.ss.usermodel.Workbook
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
+import org.assertj.core.api.Assertions.*
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.pecs.jpc.location.Location
+import uk.gov.justice.digital.hmpps.pecs.jpc.location.LocationRepository
+import uk.gov.justice.digital.hmpps.pecs.jpc.location.LocationType
+import uk.gov.justice.digital.hmpps.pecs.jpc.pricing.Supplier
+
+internal class PricesSpreadsheetTest {
+
+    private val workbook: Workbook = XSSFWorkbook()
+    private val sheet: Sheet = workbook.createSheet()
+    private val location: Location = Location(LocationType.CC, "agency id", "site name")
+    private val locationRepository: LocationRepository = mock()
+    private val spreadsheet: PricesSpreadsheet = PricesSpreadsheet(workbook, Supplier.GEOAMEY, locationRepository)
+
+    @Nested
+    inner class RecordingErrors {
+
+        private val row: Row = sheet.createRow(0)
+
+        @Test
+        fun `no errors by default`() {
+            assertThat(spreadsheet.errors).isEmpty()
+        }
+
+        @Test
+        fun `errors are recorded`() {
+            val exception: RuntimeException = RuntimeException("something went wrong")
```

Comment:
I guess you could just write `val exception = RuntimeException("something went wrong")`

Reply:
done